packages/tradershub/src/AppContent.tsx

```diff
@@ -16,7 +16,7 @@ const AppContent = () => {
             <AppContainer className={compareAccountsRoute ? 'max-w-[800px] lg:pt-0 lg:pb-20' : ''}>
                 <Router />
             </AppContainer>
-            {isEU && <EUDisclaimerMessage />}
+            {isEU && !compareAccountsRoute && <EUDisclaimerMessage />}
```

Comment:
This is very specific to a single page, isn't there any better way to handle this? 
What is the requirements on displaying the disclaimer?

Reply:
I will be going to put  {isEU && <EUDisclaimerMessage />}  in tradershubRoute itself now instead of the app level, which needs some CSS changes.

Reply:
Tradershubs route is bounded to a specific max-width controlled by `AppContainer` therefore if I wanna add the Disclaimer component  there I must add another condition to the class to override the `AppContainer` CSS

![Screenshot 2024-03-14 at 10 38 57 AM](https://github.com/binary-com/deriv-app/assets/64970259/6f14a2b3-d747-4a2d-affd-fbf72c446ca3)

I believe we can keep it like this instead of overriding the CSS maybe what do you think @mahdiyeh-deriv @matin-deriv?

Reply:
@mitra-deriv let's move the width related styles to each page, instead of adding those in the app_content level.